packages/tools/lint-package/.eslintrc.cjs

```diff
@@ -0,0 +1,13 @@
+// This is a workaround for https://github.com/eslint/eslint/issues/3458
+require('@rushstack/eslint-config/patch/modern-module-resolution');
+
+module.exports = {
+  extends: ['@kadena-dev/eslint-config/profile/lib'],
+  parserOptions: { tsconfigRootDir: __dirname },
+  rules: {
+    '@rushstack/typedef-var': 'off',
+    '@typescript-eslint/explicit-function-return-type': 'off',
+    '@typescript-eslint/naming-convention': 'off',
+    '@typescript-eslint/strict-boolean-expressions': 'off',
+  },
```

Comment:
Why are they disabled?

Reply:
They didn't spark joy.

Yet, thanks to your comment and after careful consideration we are able to remove two of them:

- Still disabled. See the screenshot below. I just doesn't make sense to type simple things that TS infers, especially in this case. Sometimes it has value for sure, such as with library code and/or when things need to be narrowed down, etc. Not here, though.
- Still disabled. Basically similar to the previous point.
- Removed. Not sure what happened, but when I removed this ESLint is complaining.
- Removed. I don't like to write `pkg.types === undefined || pkg.types === ''` when I can do `!pkg.types`. It's a good part of JavaScript, actually. But it's one occasion so I might be able to get over it.


<img width="560" alt="Screenshot 2023-07-14 at 2 55 28 PM" src="https://github.com/kadena-community/kadena.js/assets/456426/f5a1effa-10d0-4609-bc08-6b6983cf7640">